liquibase-couchbase/src/main/java/liquibase/ext/couchbase/change/MutateInChange.java

```diff
@@ -64,8 +66,10 @@ public SqlStatement[] generateStatements() {
         MutateIn mutate = buildMutate(keyspace);
         MutateInOptions mutateInOptions = buildOptions(expiry, preserveExpiry, storeSemantics);
         return new SqlStatement[] {
-                id == null ? new MutateInQueryStatement(mutate, mutateInOptions, whereCondition) :
-                        new MutateInStatement(mutate, mutateInOptions)
+                id == null && whereCondition == null ?
+                        new MutateInSqlPlusPlusQueryStatement(mutate, mutateInOptions, sqlPlusPlusQuery) :
+                        (id == null ? new MutateInQueryStatement(mutate, mutateInOptions, whereCondition) :
+                                new MutateInStatement(mutate, mutateInOptions))
```

Comment:
Consider pls changing to traditional - it will be much more clear
```suggestion
if (id == null && whereCondition == null) {
            return new SqlStatement[] {
                    new MutateInSqlPlusPlusQueryStatement(mutate, mutateInOptions, sqlPlusPlusQuery)
            };
        }
        if (id == null) {
            return new SqlStatement[] {
                    new MutateInQueryStatement(mutate, mutateInOptions, whereCondition)
            };
        }
        return new SqlStatement[] {
                new MutateInStatement(mutate, mutateInOptions)
        };
```